components/supervisor/pkg/supervisor/git.go

```diff
@@ -50,14 +53,22 @@ func (p *GitTokenProvider) GetToken(ctx context.Context, req *api.GetTokenReques
 	}
 	missing := getMissingScopes(req.Scope, scopes)
 	if len(missing) > 0 {
-		message := fmt.Sprintf("An operation requires additional permissions: %s. Please grant these on the [access control page](%s).", strings.Join(missing, ", "), p.workspaceConfig.GitpodHost+"/access-control")
-		_, err = p.notificationService.Notify(ctx, &api.NotifyRequest{
+		message := fmt.Sprintf("An operation requires additional permissions: %s.", strings.Join(missing, ", "))
+		result, err := p.notificationService.Notify(ctx, &api.NotifyRequest{
 			Level:   api.NotifyRequest_INFO,
 			Message: message,
+			Actions: []string{"Open Access Control"},
```

Comment:
maybe better `Grant Permissions`? i.e. call to action, or at least put in message for instance `Please grant permissions and try again`.

Reply:
I extended the message. I think it's good if the user understands that clicking the button does not yet grant the permissions.